services/context/response.go

```diff
@@ -15,27 +15,26 @@ type ResponseWriter interface {
 	http.Flusher
 	web_types.ResponseStatusProvider
 
-	Before(func(ResponseWriter))
-
-	Status() int // used by access logger template
-	Size() int   // used by access logger template
```

Comment:
These methods have been decoupled by `accessLoggerTmplData`, there won't be interface method call from templates anymore.